New landing page design

0tuedon opened the pull request.

0tuedon commented:
https://transcription-review-frontend-bkchgaxbj-btc-knowledge-base.vercel.app/
Latest update here the youtube iframe still left and mobile responsiveness cc: @Extheoisah  @Emmanuel-Develops

Extheoisah reviewed (changes requested):
First review pass on design only:
1. the font size is too large. we need to scale it to be normal according the to the screen aspect ratio.
2. there is an overflow of text in the cards in the 2nd section
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/60826700/73a0a8e7-6d9f-41fa-b824-0fc0e18bd55e)

3. the edges of the placeholder images are not rounded
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/60826700/9f207e77-b12b-4d0b-aa62-4d963bc87fef)

4. the containers housing the images in these card sections do not have an overflow hidden property. the edges still overflow and show. Its not clean and neat as the design. pls look into it.
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/60826700/b2eb2dbc-aba4-40dd-83a6-ce8ecd6daf04)

Aside these issues, this is a nice start👍. 
These are the obvious UI quirks i noticed. I'll leave another review on the code when these are fixed.

adamjonas commented on src/components/home/review/GreatTranscript.tsx:
```diff
@@ -0,0 +1,89 @@
+          fontSize={{ base: "0.875rem", lg: "1.75rem" }}
+          letterSpacing={"0.78px"}
+        >
+          <ListItem>Silent Payments and Alternatives</ListItem>
```
These should be links I believe

0tuedon replied:
oops, although i didn't get the links from the design.

adamjonas reviewed (commented):
Wow. What a difference. Nice work. 

![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/18909ba6-a1b3-4efa-88f6-c36bdb99afab)

The mobile footer needs sizing and arrangement work

adamjonas commented:
<img width="1152" alt="Screenshot 2023-11-08 at 2 03 54 PM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/3bd650b8-a0ca-48ca-a590-da29faa4988b">

This should be a link.

adamjonas commented:
![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/e7a227bf-d1eb-4afb-a267-8a59576fd441)

This image should crop out the admin checkboxes

adamjonas commented:
<img width="955" alt="Screenshot 2023-11-08 at 2 06 35 PM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/88ebaa49-2ae7-4548-b238-a4dd1ede4be0">

another button that should be a link

paperpsych commented:
Hi, here's my review on the design:

### The Hero Section

![hero-rev](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/9bbf54ad-8f4e-488c-851c-0a0eb1a71a26)


- We can make the wordmark bigger in size for better emphasis

![Screenshot 2023-11-09 015710](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/f606d502-dfb0-4262-8810-ae1164b504a2)

- A smaller headline size here, similar to the size figma design and please format it in a manner such that it only uses 03 lines too like shown in Figma (lmk if the following snippet helps with the resizing)

![Screenshot 2023-11-09 015426](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/e83ceebb-dc01-4353-af23-d37f1117a605)

- A smaller sub-headline size so that it fits in one screen with the headline and the other elements

![Screenshot 2023-11-09 020009](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/72ba1830-6bc3-4a70-8127-221034d1104c)

- We should make the CTA buttons bigger too

![Screenshot 2023-11-09 020537](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/9bd75096-da37-4f1f-8643-82e587b0a14f)

- The faded orange grid in the background should have smaller grids

- And add the required animations to the CTAs and the video element

https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/af29d119-1a9f-4c48-b171-f7066b590fd0

The CTAs (as mentioned above) are not links, they should be. I couldn't find a "copy link" option for them in the original site so I couldn't add them in the design. Here's the function of the CTAs in the hero section:

- "Get Started" is what takes the user to the Github Sign-In page that will ask them to connect with btctranscripts. The "Sign-In" in the nav bar of the original site does this.
- "Tell me More↓" scrolls down to the next section of the landing page. The "Get started" button does this on the original site.

Overall, the design has came out really well, I appreciate the effort that has been put into developing the site so far :)

P.S. Let me know if the code snippets are helpful or else I'll refrain from using them in the comments. Or else you can directly see them in Figma too !!

Emmanuel-Develops commented:
Looks good!

Some border, radius and padding quirks 
<img width="1397" alt="Screenshot 2023-11-09 at 9 14 03 AM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/62019510/922e08ee-b094-42dd-a4c9-81c5ebc8ed34">
<img width="1120" alt="Screenshot 2023-11-09 at 9 10 29 AM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/62019510/7ab0d107-2154-4662-b341-992d917d8452">

0tuedon commented:
Working on this issues , Thanks @everyone

adamjonas commented:
Getting a double header on mobile ![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/f0013004-d63c-4299-b70c-6bca9e2bf96b)

adamjonas commented:
Also a double footer 

![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/118638f3-6c06-419a-b4d9-5f70642012ac)

0tuedon commented:
> Getting a double header on mobile ![image](https://user-images.githubusercontent.com/755825/282080143-f0013004-d63c-4299-b70c-6bca9e2bf96b.png)

oh right fixing it.

adamjonas commented:
The spacing and sizing in the footer needs adjustments. ![image](https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/f54d1a08-34f9-40e2-a242-e8a885ebf647)

The top text seems totally disconnected from the bottom text.

0tuedon commented:
.

adamjonas commented:
For mobile: 
1. Cut padding on "what you'll need" section
2. These elements have a vertical scroll in them on mobile: <img width="371" alt="Screenshot 2023-11-10 at 3 44 21 PM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/9d453a13-593e-454e-aaca-2fdfe173dea2">
<img width="256" alt="Screenshot 2023-11-10 at 3 43 59 PM" src="https://github.com/bitcointranscripts/transcription-review-front-end/assets/755825/d2190ef5-d721-464d-901f-62ef315fb329">
3. Font size consistency in step 3

@aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.

0tuedon commented:
> For mobile:
> 
> 1. Cut padding on "what you'll need" section
> 2. These elements have a vertical scroll in them on mobile:
> 
> 3. Font size consistency in step 3
> 
> @aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.



> For mobile:
> 
> 1. Cut padding on "what you'll need" section
> 2. These elements have a vertical scroll in them on mobile:
> 
> 
> @aassoiants does a review [here](https://drive.google.com/file/d/1j5XWv3HnyiNJkw4L2M8iDijxFpyi262E/view). Let's address the above pieces but I think we an wait on the others.

the lastest commit currently addresses these issues.

aassoiants commented:
Thanks team!

Emmanuel-Develops reviewed (dismissed):
All looks good!
rebase with updated dev branch and LGTM

paperpsych commented:
Please reduce the size of the white boxes and the font size of the headings and body text inside the boxes.

![Screenshot 2023-11-13 120209](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/3f0f9df0-bddd-424c-867e-31c3d9320598)
![Screenshot 2023-11-13 121224](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/0f510395-f38f-48fe-b702-2566de9c437a)

Extheoisah commented on src/components/home/CarouselCards.tsx:
```diff
@@ -0,0 +1,31 @@
+          <CarouseCardSlide {...content} />
+        </SplideSlide>
+      ))}
+      {/* <SplideSlide></SplideSlide> */}
```
```suggestion
```
remove comment

Extheoisah commented on src/components/home/NeedSection.tsx:
```diff
@@ -0,0 +1,127 @@
+            </Text>
+          </Flex>
+
+          {/*  */}
```
remove
```suggestion
```

Extheoisah commented on src/components/home/StepLayout.tsx:
```diff
@@ -0,0 +1,106 @@
+              </Box>
+            }
+            fontSize={{ base: "12px", lg: "18px", "2xl": "24px" }}
+            // size={{ base: "md", lg: "lg" }}
```
```suggestion
```

Extheoisah commented on src/components/home/steps/Step1.tsx:
```diff
@@ -0,0 +1,187 @@
+      setCurrentIndex((prevIndex) =>
+        prevIndex === images.length - 1 ? 0 : prevIndex + 1
+      );
+    }, 10000); // Change slide every 3 seconds (3000 milliseconds)
```
```suggestion
    }, 10000); // Change slide every 10 seconds
```

Extheoisah commented on src/components/home/steps/Step1.tsx:
```diff
@@ -0,0 +1,187 @@
+      link={"https://www.youtube.com/watch?v=YNIFm0QFAuA&t=2m16s"}
+    >
+      <Flex flexDir={"column"} gap={10}>
+        {/* <Flex></Flex> */}
```
```suggestion
```

Extheoisah commented on src/components/home/steps/Step3.tsx:
```diff
@@ -0,0 +1,175 @@
+      setCurrentIndex((prevIndex) =>
+        prevIndex === images.length - 1 ? 0 : prevIndex + 1
+      );
+    }, 10000); // Change slide every 3 seconds (3000 milliseconds)
```
```suggestion
    }, 10000); // Change slide every 10 seconds
```

paperpsych commented:
We'll need the sizing of text in this whole section to be reduced too.

![Screenshot (926)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/429c5991-30a1-4656-b5c1-3fa10c045da1)
![Screenshot (927)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/97ddb4c8-bb7f-4d18-9fb0-117c010e8387)

---

Also, please change the orange on the whole website from "FFA500" to "F7931A".

---

Here the 2nd half of the "Authorize" step is staying dormant.

![Screenshot 2023-11-14 185510](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/3545321f-a18f-4d9a-9e42-bacb77c1807e)

This is how it should look. And please adjust the vertical gap between the heading and the body to match the height & alignment of all the 03 parts of texts.

![Screenshot 2023-11-14 185437](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/5b2134a2-9313-4c4b-868e-a576bdfa7ffe)

---
The "Watch tutorial" buttons should have more curve radius and more padding on top and bottom.

![Screenshot 2023-11-14 190135](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/4e68de8d-4569-4fa1-aac3-27510e665e99)
![Screenshot 2023-11-14 190156](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/9693670f-c06f-4189-9b31-bec34f12574b)

paperpsych commented:
[Here's the link to the prototype for reference](https://www.figma.com/proto/PK9XQob6bVMw5Q0rJhYbu2/BTC-Transcript-Review?page-id=0%3A1&type=design&node-id=155-98&viewport=-1243%2C281%2C0.52&t=8Cmq8zkS9KibC809-1&scaling=scale-down-width&mode=design)

**Hero Section**

- Wordmark (logo text) slightly bigger
- Subheadline smaller in size
- Smaller YouTube video element
- Buttons seem to have unequal top and bottom padding
- "Tell me more" button's arrow needs to be smaller to match the button text's height

![Screenshot (929)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/77c242d8-f5cf-4810-9287-94f48d298df8)

---

**Overall text formatting on the website**

- Please match the text's line breaks
- Match the spacing of elements from the edge and other elements
- Match button hover animations

---

**Why edit transcripts?**

- Smaller font sizes
- The image's size needs to more rectangular instead of square
- And more gap from the left edge and less gap between the image and the text content

![Screenshot (930)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/bd42541b-f56b-4553-a799-3f3c52c90074)

---

**How it works**

- Text alignments

![Group 4669](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/532f0f1d-bbde-4949-bb05-a2d783c59b6f)

- Step 03's icon should be changed to the tick mark one
- "After Submission" text block should be of the PolySans typeface

---

**Review Process Tips**

- Smaller overall size of the section
- [Use this image](https://drive.google.com/file/d/1voXTa0ALPVcT0KNBKLEq3SOvFxpgOByP/view?usp=drive_link) for the split screen section

---

**Primary CTA and Footer**

- Changes in CTA button's size, accompanying copy's font size
- Gap between the primary CTA section, the review process section, and footer
- _Submit feedback_ side in Aeonik Fono typeface and _Bitcoin Dev Project_ side in PolySans typeface (and both need to be closer to their edges)

![Screenshot (931)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/217981a5-795c-42f6-aeac-816b61051307)

![Screenshot (932)](https://github.com/bitcointranscripts/transcription-review-front-end/assets/149538435/89de4b6f-15f7-4fb8-8f8a-eef2fdcd6237)

0tuedon commented:
> We'll need the sizing of text in this whole section to be reduced too.
> 

> ![Screenshot 2023-11-14 190135](https://user-images.githubusercontent.com/149538435/282790127-4e68de8d-4569-4fa1-aac3-27510e665e99.png) ![Screenshot 2023-11-14 190156](https://user-images.githubusercontent.com/149538435/282790163-9693670f-c06f-4189-9b31-bec34f12574b.png)

All this issues is being referenced in #224
